### ISSUE TITLE
Can't watch catchup

### ISSUE THREAD

**Issue by dhruv-glidetech** [kiranreddyrebel/kiranreddyrebel.github.io#8]
![IMG_20240803_135056](https://github.com/user-attachments/assets/23317e56-3b91-4435-ae9d-6c64a8acd1dd)
![IMG_20240803_135023](https://github.com/user-attachments/assets/51dc36ff-bc5d-4fb2-bc9a-a45329cddf70)

Thanks in advance.

**Comment by kiranreddyrebel** [comment]
Its working. May I know which version of plugin you have installed.

There is ver. 2.3.12 version available. Check it.

Remove old one completely and clean install kodi.

Then install.

**Comment by Rajkalithas** [comment]
![17227482295344872269216523645254](https://github.com/user-attachments/assets/514fdc6e-6bdc-413f-a05b-bd2ee128be09)
I try to add your repository 
But following add ons not available 
![17227482868391998606184632724339](https://github.com/user-attachments/assets/b90d1bbc-cbc9-4f49-b85b-7645e0757617)
The i clicked on . It's showing the dependency could not be satisfied

**Comment by dhruv-glidetech** [comment]
> Its working. May I know which version of plugin you have installed.
> 
> There is ver. 2.3.12 version available. Check it.
> 
> Remove old one completely and clean install kodi.
> 
> Then install.
Catchup means... channel's previous programs

Same issue....

**Comment by dhruv-glidetech** [comment]
> Its working. May I know which version of plugin you have installed.
> 
> There is ver. 2.3.12 version available. Check it.
> 
> Remove old one completely and clean install kodi.
> 
> Then install.

Catchup means.... previous programs.

Same issue with mentioned version.

Any solution?
Thanks in advance.

**Comment by technofreak24** [comment]
**Catchup** feature works now after updating by some developer. 401 Error is gone. 

![Screenshot_20240820-012531](https://github.com/user-attachments/assets/39d678d3-23e0-4cf0-8adf-72c69786733c)

![Screenshot_20240820-012607](https://github.com/user-attachments/assets/e8c6c21b-5053-440d-87c7-b0d46ab70cc0)

**Comment by kiranreddyrebel** [comment]
> From where I can get newer version?

https://github.com/kiranreddyrebel/kiranreddyrebel.github.io/tree/main

**Comment by dhruv-glidetech** [comment]
Version 2.3.12?

**Comment by GTTech2305** [comment]
Catchup is stop from jio it'sself for many channel's

**Comment by dhruv-glidetech** [comment]
Ohh okay NP, but its working on JioTV mobile application thats why I asked.

**Comment by GTTech2305** [comment]
Yes work but only some channel you can check app also zee network not have catchup options

**Comment by dhruv-glidetech** [comment]
Yes, For colors its working in app but in addon not.

**Comment by kiranreddyrebel** [comment]
Colors working bro. Try reset the iptv cache and restart kodi. All channels working.

**Comment by dhruv-glidetech** [comment]
Yes, Colors Channel is working but its catchup not working.

**Comment by Mfstick** [comment]
@kiranreddyrebel @dhruv-glidetech @GTTech2305 
Any update on catchUp issue. HistoryTV working only on live and not catchUp.

**Comment by Poulocean** [comment]
Still catchup not working.

**Comment by kiranreddyrebel** [comment]
you should enable the option "Generate playlist on startup" inside the addon's configure (settings). Try it.

**Comment by Mfstick** [comment]
@kiranreddyrebel  catchup still not working even after adding startup on playlist.
It throws 401 error. I tried history tv. 
Note: catchup flow works on original jiotv app. So functionality is there. May be api flow changes. Could you please fix this. It would be more helpful. Thank you!
